src/identity/interaction/email-password/handler/LoginHandler.ts

```diff
@@ -23,6 +24,10 @@ export class LoginHandler extends InteractionHandler {
     const { email, password, remember } = await this.parseInput(operation);
     // Try to log in, will error if email/password combination is invalid
     const webId = await this.accountStore.authenticate(email, password);
+    const settings = await this.accountStore.getSettings(webId);
+    if (!settings.useIdp) {
+      throw new BadRequestHttpError('This account is not registered for identification');
```

Comment:
I find the error message hard to understand, but couldn't make a suggestion as I don't know what it means exactly. Something like "This server is not an identity provider for the account xxx"?

Reply:
That is what it means yes, there is an account but it is only registered for pod management, not for IDP usage.

Reply:
Okay, let's update the error message and also add a comment at that location in the code, explaining the above.